.github/workflows/gha.yml

```diff
@@ -115,7 +115,7 @@ jobs:
         restore-keys: ccache-${{ env.CC_BUILD_ID }}-static-build-
 
     - name: Install basic dependencies
-      run: sudo apt install autoconf ccache libnuma-dev
+      run: sudo apt install autoconf ccache libnuma-dev pandoc
```

Comment:
I think this is not the job that is failing.

The one that's failing has `name: Docker build [xxx]`